### ISSUE TITLE
Compile with --use_strict flag

### ISSUE THREAD

**Issue by tresdin** [jaredallard/nexe#168]
I'm using some ES6 features that require strict mode. Normally, I would run my app with this command:
```
node --use_strict /home/project/app.js
```
How can I compile my entire app with the `--use_strict` flag?

**Comment by jaredallard** [comment]
Have you tried `"use strict";` in your scripts that require it?

**Comment by tresdin** [comment]
@jaredallard It's quite inconvenient this way since I will need to write an extra function that programmatically includes `"use strict";` on top of each file. Is `nexe` going to support this flag?

**Comment by jaredallard** [comment]
@tresdin Well, I'm trying to think of a way to do this. I'll see what I can do!

**Comment by jaredallard** [comment]
Hey! So been working on this today.

So, I thought of how NW.js may handle command lines, which is probably via the source code. So, I took a look at the node source. Found `--use_strict` is a v8 flag (duh) and took a look at v8 source. Got a semi understanding of how v8 handles flags, wrote some hacky c++ code, and it seems to have worked. Will work on making it part of a monkey patch procedure, then seeing about backwards compatability (as it is variable for other versions, I'm sure).

Source (in case you want to manually work that for a bit):

(node-src)/deps/v8/src/flags.cc

```cpp
  /* 574-578 */
  /* custom args. */
  const char* nexevargs = "--use_strict";
  int nexevargslen = strlen(nexevargs);

  SetFlagsFromString(nexevargs, nexevargslen);
```
